network/definition/data/templates/messages.yaml

```diff
@@ -31,3 +31,9 @@ messages:
     signals:
       udsPayload:
         template: udsPayload
+  buildInfo:
+    signals:
+      crcRepo:
+        template: crc32
```

Comment:
just publish the commit hash. `appGitHash` and then in the bl you could do `bootloaderGitHash`